tools/tracevis/lib/trace/criticalPath.js

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2016 LinkedIn, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+function ExecutionUnit(id, start, end) {
```

Comment:
At the top, could you add a comment that describes description of how critical path is calculated?